### ISSUE TITLE
Issue in return-in-computed-property & require-render-return

### ISSUE THREAD

**Issue by armano2** [vuejs/eslint-plugin-vue#130]
I tried to test new rules in one of my big projects and i have few false positive errors in case when there is CodePath/Function inside with no return.

Affected rules:
- `return-in-computed-property` 
- `require-render-return`

Example of false positive error:
```js
export default {
  computed: {
    foo () {
      const options = []
      this.matches.forEach((match) => {
        options.push(match)
      })
      return options
    }
  }
}
```
